MirrorProvider/MirrorProvider.Linux/LinuxFileSystemVirtualizer.cs

```diff
@@ -12,10 +12,8 @@ public class LinuxFileSystemVirtualizer : FileSystemVirtualizer
 
         public override bool TryConvertVirtualizationRoot(string directory, out string error)
         {
-            Result result = VirtualizationInstance.ConvertDirectoryToVirtualizationRoot(directory);
-
-            error = result.ToString();
-            return result == Result.Success;
+            error = Result.Success.ToString();
```

Comment:
For consistency, this should probably be `error = null`

Reply:
Gotcha, thanks!